index.js

```diff
@@ -0,0 +1,78 @@
+console.log("Ex1-----------------");
+const numberOperation = (number) => {
+  if (number % 2 !== 0) {
+    return number * 2;
+  } else {
+    return number - 2;
+  }
+}
+console.log(numberOperation(3));
+
+console.log("Ex2-----------------");
+let numArray = [2, 3, 4, 5, 6, 7, 8, 9, 10];
+const savePrimes = (array) => {
+  let arrayPrimeNumbers = Array();
+  arrayPrimeNumbers = array.filter((number) => {
+    for (let i = 2; i <= Math.sqrt(number); i++) {
+      if (number % i === 0) return false;
+    }
+    return true;
+  });
+  return arrayPrimeNumbers;
+}
+const primeNumbers = savePrimes(numArray);
+for (let i = 0; i < primeNumbers.length; i++) {
+  console.log(primeNumbers[i]);
```

Comment:
Aici era ok si sa faci direct console.log(primeNumbers), e suficient folosirea for-ului doar odata in task :D